src/main/java/org/openmrs/uitestframework/page/Page.java

```diff
@@ -345,10 +345,11 @@ public Boolean containsText(String text) {
 
     public List<String> getValidationErrors() {
         List<String> validationErrors = new ArrayList<String>();
-        for (WebElement webElement : driver.findElements(By.className("error"))) {
```

Comment:
i can see the field with class error, are u sure you are pointing to the right element, there are lots on field-error elements on the same page

Reply:
have you tried checking, use your chrome tools on demo or qa instances and check, i can see one .error element and i wanna be sure we know difference between it and these field errors!

Reply:
> have you tried checking, use your chrome tools on demo or qa instances and check, i can see one .error element and i wanna be sure we know difference between it and these field errors!

ok am gona test this and get back here